doc/source/testing.rst

```diff
@@ -66,13 +80,19 @@ Run all functional tests.
 
 .. code-block:: bash
 
-    $ tox -t functional
+    $ tox -e $(tox -l | grep functional | paste -d, -s -)
+
+
+Run all functional tests for a specific version of Python and Ansible (here
+Python 3.7 and Ansible 2.7).
+
+    $ tox -e py37-ansible27-unit
 
 Run all functional tests targeting the docker driver.
 
 .. code-block:: bash
 
-    $ tox -t functional -- -v -k docker
+    $ tox -e $(tox -l | grep functional | paste -d, -s -) -- -v -k docker
```

Comment:
```suggestion
    $ tox -e functional -- -v -k docker
```

Reply:
Sorry but it would execute the tests only on the current python version, see https://github.com/ansible/molecule/issues/1764#issuecomment-465992799